node/core/dispute-coordinator/src/tests.rs

```diff
@@ -757,8 +796,9 @@ fn approval_vote_import_works() {
 			handle_approval_vote_request(&mut virtual_overseer, &candidate_hash1, approval_votes)
 				.await;
 
-			// Participation has to fail, otherwise the dispute will be confirmed.
-			participation_missing_availability(&mut virtual_overseer).await;
+			// Participation has to fail here, otherwise the dispute will be confirmed. However
+			// participation won't happen at all because the dispute is neither backed, not confirmed
```

Comment:
I think we remove the comment about failing here. If we are not participating, nothing fails. Just saying that we won't participate at this time because .... would make more sense I think

Reply:
I agree with 'participation refraining' this will be more confusing than informative.